org.pwsafe.pwsafe.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Rony Shapiro <ronys@pwsafe.org> -->
+<!--
+EmailAddress: ronys@pwsafe.org
+SentUpstream: 2024-01-25
+-->
+<application>
+  <id type="desktop">org.pwsafe.pwsafe</id>
+  <name>Password Safe</name>
+  <summary>Simple and Secure Password Manager</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Artistic License 2.0</project_license>
+  <description>
+    <p>
+        Password Safe is cross-platform password manager. It supports
+        two-factor authentication with Yubikey.
+    </p>
+  </description>
+  <releases>
+    <release version="v1.18.2" date="2024-01-23">
+      <description>
+        <p>The latest upstream tag.</p>
+      </description>
+    </release>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.pwsafe.org/images/treeview.png</image>
```

Comment:
Can there be a bigger screenshot and one that is on Linux?

Reply:
@ronys, I have created print screen from Ubuntu 23.10, but it should look the same on any Linux distro. See if you like the picture and upload it like to www.pwsafe.org web site.

![PasswordSafe_on_Linux](https://github.com/flathub/flathub/assets/10895030/f53d6643-b519-4705-8563-c1a48420243e)

I just created few "groups" and "entries" and added right menu to display how easy is to add new entry.

ADDITIONALLY:
@ronys, I think there should be written somewhere that Password Safe is cross platform: Windows, Linux and MacOS. This is in my humble opinion the main strength of the program. Few years ago, when I was searching for password manager tool, this was the only decent cross-platform tool I have found (because I use Windows at my daily job and Ubuntu at home).